bootstrap.sh

```diff
@@ -35,39 +35,47 @@ then
     then
         echo "Downloading clang-tidy and clang-format to bin directory"
```

Comment:
Output of dotnet --info:
.NET Command Line Tools (1.0.0-preview2-003121)

Product Information:
 Version:            1.0.0-preview2-003121
 Commit SHA-1 hash:  1e9d529bc5

Runtime Environment:
 OS Name:     centos
 OS Version:  7
 OS Platform: Linux
 RID:         centos.7-x64


---
In reply to: [75393080](https://github.com/dotnet/jitutils/pull/28#discussion_r75393080) [](ancestors = 75393080)